mobile/src/Student/components/Answers/Answer/index.js

```diff
@@ -1,129 +1,200 @@
-import React from 'react'
-import { StyleSheet, Text, View, Image } from 'react-native'
-import { scale, verticalScale } from 'react-native-size-matters'
+import React, { useState, useRef } from "react";
+import { StyleSheet, Text, View, Image, Animated } from "react-native";
+import { scale } from "react-native-size-matters";
+import LinearGradient from "react-native-linear-gradient";
+import * as Progress from "react-native-progress";
```

Comment:
nit: For consistency would be nice not to have `;` at the end.

Reply:
👍🏻